packages/x-utils/src/services/session.service.ts

```diff
@@ -21,7 +25,9 @@ export class DefaultSessionService implements SessionService {
   public static instance: SessionService = new DefaultSessionService();
 
   public constructor(
-    protected storageService: StorageService = new BrowserStorageService(localStorage, 'x'),
+    protected storageService: StorageService = typeof localStorage !== 'undefined'
+      ? new BrowserStorageService(localStorage, 'x')
+      : new InMemoryStorageService(),
```

Comment:
I'd like to remove this singleton and the default parameters so it is not coupled to the browser. The service instantiation should be done somewhere else in x-components. Initially the idea we had with services was to add them to the x-module. But for this PR I think it is fine. We can work this in the future.